Makefile

```diff
@@ -34,3 +34,9 @@ test: venv
 .PHONY: venv
 venv:
 	test -d $(VENV_PATH) || python3 -m venv $(VENV_PATH)
+
+.PHONY: performance
+performance: venv install
```

Comment:
Does this just return the time it takes to complete??

Reply:
It will run a general test case and then produce a `snakeviz` runtime page (the same one we were looking at earlier today).